tests/test_utils.py

```diff
@@ -113,9 +113,9 @@ def test_invalid_ssObjectID_to_objID_and_objID_to_ssObjectID(self):
         """Convert between ssObjectIDs and MPC packed designations
         """
         allowed_strings = [ch for ch in 'ABCDEFGHIJKMNOPQRSTUVWXYZ0123456789 -'] \
-                + ['A' * i for i in range(2, 8)] + ['Z' * i for i in range(2, 8)] \
-                + ['J95X00A', 'J95X01L', 'J95F13B', 'J98SA8Q', 'J98SC7V', 'J98SG2S'] \
-                + ['K99AJ3Z', 'K08Aa0A', 'K07Tf8A', 'PLS2040', 'T1S3138', 'T2S1010', 'T3S4101']
+            + ['A' * i for i in range(2, 8)] + ['Z' * i for i in range(2, 8)] \
+            + ['J95X00A', 'J95X01L', 'J95F13B', 'J98SA8Q', 'J98SC7V', 'J98SG2S'] \
+            + ['K99AJ3Z', 'K08Aa0A', 'K07Tf8A', 'PLS2040', 'T1S3138', 'T2S1010', 'T3S4101']
```

Comment:
I assume this breaks `flake8`, so this commit should have been part of "adding unit tests"